File: samples/CS/17-QuickAccessToolbar/Form1.cs
using System;
using System.Windows.Forms;
using WinForms.Ribbon;
using System.IO;

namespace _17_QuickAccessToolbar
{
    public partial class Form1 : Form
    {
        private RibbonItems _ribbonItems;

        public Form1()
        {
            InitializeComponent();
            _ribbonItems = new RibbonItems(_ribbon);
            _ribbonItems.Init();

        }
    }
}

File: samples/VB/17-QuickAccessToolbar/Form1.vb
Imports WinForms.Ribbon
Imports System.IO

Namespace _17_QuickAccessToolbar

	Partial Public Class Form1
		Inherits Form
		Private _ribbonItems As RibbonItems

		Private _stream As Stream

		Public Sub New()
			InitializeComponent()
			_ribbonItems = New RibbonItems(_ribbon)
			_ribbonItems.Init()
		End Sub
	End Class
End Namespace
